### DISCUSSION TITLE
How to build jpgouin/openldap:2.6.8-fix

### DISCUSSION THREAD

**Discussion by A-little-bit-of-data** [jp-gouin/helm-openldap#201] [Q&A]
Hello, I want to know how to build` jpgouin/openldap:2.6.8-fix `and want to see this dockerfile. I need to install some commands in the container and also want to know the difference between it and the same version image of bitnami. I also have a question, are the Feature Release and LTS Release of openldap applicable to this helm package?

**Comment by jp-gouin** [comment]
Hi, It's now available under https://github.com/jp-gouin/openldap-container.git

**Reply by jp-gouin** [reply]
The fix itself is documented [here ](https://github.com/bitnami/containers/pull/53960)
It was reverted for some backward compatibility issue

**Reply by A-little-bit-of-data** [reply]
Thank you very much, this is great！I would like to ask, the latest version of LTS Release of openldap should also support this helm version

**Reply by jp-gouin** [reply]
Yes it's possible. The fix should be backported in 2.5. The 2.5 folder should be (re)copied from Bitnami.
